brainiak/utils/fmrisim.py

```diff
@@ -1382,7 +1475,7 @@ def generate_noise_volume(dimensions,
                                                              1)
     temporal_noise = temporal_noise - (temporal_noise_mean - spatial_noise)
```

Comment:
Since I multiply these time series, it is critical that they have a mean of zero to ensure that the variance of the time course is the only thing I am manipulating. Hence it is necessary that each time series is mean zero